app/presenters/tree_node/miq_product_feature.rb

```diff
@@ -0,0 +1,54 @@
+module TreeNode
+  # Current assumption is that this builder would only be used in Rbac Features tree
+  #
+  class MiqProductFeature < TreeNode::Menu::Node
+    set_attribute(:key)     { "#{@options[:node_id_prefix]}__#{@object.identifier}" }
+    set_attribute(:title)   { _(@object.name) }
+    set_attribute(:tooltip) { _(@object.description) || _(@object.name) }
+
+    set_attribute(:icon) do
+      case @object.feature_type
+      when 'view'     then 'fa fa-search'
+      when 'control'  then 'fa fa-shield'
+      when 'admin'    then 'pficon pficon-edit'
+      else
+        'pficon pficon-folder-close'
+      end
+    end
+
+    def select_state
+      self_selected? || parent_selected? || select_by_children_selected
+    end
+
+    def children
+      ::MiqProductFeature.obj_feature_all_children(@object.identifier).map do |child|
+        TreeNode::MiqProductFeature.new(child, key, @options)
+      end
+    end
+
+    private
+
+    def self_selected?
+      @options[:features].include?(identifier_without_accords)
+    end
+
+    def identifier_without_accords
```

Comment:
One-liner used in a single place...please put it back and add a comment if necessary.